modules/web/src/assets/css/material/_main.scss

```diff
@@ -447,6 +447,10 @@ mat-form-field {
   }
 }
 
+.static-labels-values mat-form-field.km-node-tags .mat-mdc-text-field-wrapper .mdc-floating-label--float-above{
+  top: 30px !important;
+}
```

Comment:
Why do we need this? Can you please share a screenshot?

Reply:
this is a pic without this
![image](https://github.com/user-attachments/assets/a4096f30-85aa-431b-8398-e5c5f255778c)

but i check it again and maybe 28px would look better
![image](https://github.com/user-attachments/assets/aa7b42b1-f2a9-49f9-8033-0088efc408be)